### ISSUE TITLE
Cannot launch shiny app

### ISSUE THREAD

**Issue by olivertam** [ergonyc/omicser#32]
Omicser was installed via `devtools::github("ergonyc/omicser")`:
``` R
✓  checking for file ‘/private/var/folders/s5/rcjgwwvd6sx6vhm1q0c7z7q40000gn/T/RtmpcGWCBY/remotes6cc56ee57ed2/ergonyc-omicser-bba4b88/DESCRIPTION’ ...
─  preparing ‘omicser’:
✓  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
   Omitted ‘LazyData’ from DESCRIPTION
─  building ‘omicser_0.0.9.17092021.tar.gz’

* installing *source* package ‘omicser’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (omicser)
> library(omicser)
> omicser::launchApp()
Error: 'launchApp' is not an exported object from 'namespace:omicser'
```
``` R
> sessionInfo()
R version 4.1.1 (2021-08-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] omicser_0.0.9.17092021 reticulate_1.22       

loaded via a namespace (and not attached):
 [1] httr_1.4.2               tidyr_1.1.4              pkgload_1.2.2            viridisLite_0.4.0        jsonlite_1.7.2          
 [6] shiny_1.7.0              yaml_2.2.1               remotes_2.4.1            sessioninfo_1.1.1        pillar_1.6.3            
[11] lattice_0.20-44          glue_1.4.2               digest_0.6.28            RColorBrewer_1.1-2       promises_1.2.0.1        
[16] colorspace_2.0-2         htmltools_0.5.2          httpuv_1.6.3             Matrix_1.3-4             pkgconfig_2.0.3         
[21] devtools_2.4.2           config_0.3.1             purrr_0.3.4              xtable_1.8-4             scales_1.1.1            
[26] processx_3.5.2           later_1.3.0              tibble_3.1.4             generics_0.1.0           ggplot2_3.3.5           
[31] usethis_2.0.1            ellipsis_0.3.2           DT_0.19                  cachem_1.0.6             withr_2.4.2             
[36] lazyeval_0.2.2           RcppTOML_0.1.7           cli_3.0.1                magrittr_2.0.1           crayon_1.4.1            
[41] mime_0.12                memoise_2.0.0            ps_1.6.0                 golem_0.3.1              fs_1.5.0                
[46] dockerfiler_0.1.4        fansi_0.5.0              xml2_1.3.2               pkgbuild_1.2.0           shinydashboard_0.7.1    
[51] tools_4.1.1              data.table_1.14.2        prettyunits_1.1.1        lifecycle_1.0.1          stringr_1.4.0           
[56] plotly_4.9.4.1           munsell_0.5.0            configr_0.3.5            callr_3.7.0              compiler_4.1.1          
[61] rlang_0.4.11             grid_4.1.1               attempt_0.3.1            shinydashboardPlus_2.0.3 rstudioapi_0.13         
[66] htmlwidgets_1.5.4        testthat_3.0.4           gtable_0.3.0             roxygen2_7.1.2           curl_4.3.2              
[71] R6_2.5.1                 ini_0.3.1                knitr_1.35               dplyr_1.0.7              fastmap_1.1.0           
[76] utf8_1.2.2               rprojroot_2.0.2          desc_1.4.0               stringi_1.7.4            Rcpp_1.0.7              
[81] vctrs_0.3.8              png_0.1-7                tidyselect_1.1.1         xfun_0.26

**Comment by h-leonard** [comment]
Ran into the same issue, the `omicser::launchApp()` results in an error, but the correct way to launch the app is in the Quickstart documents [(https://ergonyc.github.io/omicser/articles/05_browsing.html)] - `omicser::run_app(options = list(launch.browser = TRUE))`

The second way seemed to worked for me, the documentation for `omicser::launchApp()` should be updated

**Comment by olivertam** [comment]
I can confirm that the second approach works.
Thanks.

**Comment by ergonyc** [comment]
fixed with #35
